site/source/docs/creating_plugins.md

```diff
@@ -22,17 +22,14 @@ module.exports = function(embark) {
 }
 ```
 
-The `embark` object provides plenty of APIs to extend different functionalities of Embark, which can be found in the [Plugin Api Reference](plugin_reference.html). 
+The `embark` object provides plenty of APIs to extend different functionalities of Embark, which can be found in the [Plugin Api Reference](plugin_reference.html).
 
 ## Usecase examples
 
 Since the Plugin API surface is rather big, here are some usecase examples to sparkle some inspiration:
 
 * Adding a Smart Contract file to the list of source files to be watched so they can be used by other Smart Contracts with `addContractFile()`
-* Adding a Smart Contract configuration using `registerContractConfiguration()` (goes well with `addContractFile()`)
```

Comment:
Having the possibility to add contracts and contract configs is useful. Should we expose the events for that instead?